addon/serializers/embedded-records-mixin.js

```diff
@@ -198,13 +198,16 @@ export default Ember.Mixin.create({
     var includeIds = this.hasSerializeIdsOption(attr);
     var includeRecords = this.hasSerializeRecordsOption(attr);
     var embeddedSnapshot = snapshot.belongsTo(attr);
-    var key;
     if (includeIds) {
```

Comment:
Hmm, the logic inside `if (includeIds)` basically is the same compared to the one in [`super.serializeBelongsTo`](https://github.com/emberjs/data/blob/db49f961653246318f67f86d0427c929810fae44/addon/serializers/json.js#L1184-L1202), so it looks like this should definitely be DRY'ed up...